nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/hosted/ConanHostedRecipeTest.java

```diff
@@ -142,4 +151,24 @@ public void canMatchOnConanPackageZipUploadPackage() {
     assertThat(matcherState.getTokens().get("version"), is(equalTo("2.1.1")));
     assertThat(matcherState.getTokens().get("sha"), is(equalTo("5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9")));
   }
+
+  @Test
+  public void hostedEnabledNexusConanHostedEnabledIsTrue() {
+    System.setProperty(HOSTED_ENABLED_PROPERTY, "true");
+    ConanHostedRecipe conanHostedRecipe = new ConanHostedRecipe(new HostedType(), new ConanFormat(), null);
+    assertThat(conanHostedRecipe.isFeatureEnabled(), is(true));
+  }
+
+  @Test
+  public void hostedDisabledNexusConanHostedEnabledIsFalse() {
+    System.setProperty(HOSTED_ENABLED_PROPERTY, "false");
+    ConanHostedRecipe conanHostedRecipe = new ConanHostedRecipe(new HostedType(), new ConanFormat(), null);
+    assertThat(conanHostedRecipe.isFeatureEnabled(), is(false));
+  }
+
+  @Test
+  public void hostedDisabledNexusConanHostedEnabledIsNotSet() {
+    ConanHostedRecipe conanHostedRecipe = new ConanHostedRecipe(new HostedType(), new ConanFormat(), null);
+    assertThat(conanHostedRecipe.isFeatureEnabled(), is(false));
+  }
```

Comment:
maybe make sense to add new line in the end of this file

Reply:
Fixed